### PR TITLE
Create mehdibo/oauth2-fortytwo recipe

### PR DESCRIPTION
**Pull request by mehdibo**
| Q             | A
| ------------- | ---
| License       | MIT
| Packagist     | [mehdibo/oauth2-fortytwo](https://packagist.org/packages/mehdibo/oauth2-fortytwo)

### CODE REVIEW

**Comment by mehdibo** [comment]
I'm not sure what's the problem, or does it only need a review?

File: mehdibo/oauth2-fortytwo/1.0/manifest.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "env": {
+        "OAUTH_FT_ID": "APPLICATION_ID",
+        "OAUTH_FT_SECRET": "APPLICATION_SECRET"
```
Comment (Nyholm) [inline mehdibo/oauth2-fortytwo/1.0/manifest.json:4]: I dont see where these are used.

**Review by Nyholm — CHANGES_REQUESTED** [review]
Thank you.

It seams like your recipe contains just two env var and they are not used in the recipe or your package. 

I suggest close this PR.

File: mehdibo/oauth2-fortytwo/1.0/manifest.json (hunk 1 above)
Comment (Nyholm) [reply mehdibo/oauth2-fortytwo/1.0/manifest.json:4]: OKey. I see. 

I think we should add a recipe for the knpy/oauth2-client instead then

Comment (mehdibo) [reply mehdibo/oauth2-fortytwo/1.0/manifest.json:4]: Ok I will try to work on one once I got sometime, I will close this PR

**Pull request closed without merging**